keel/api/v1/auth/views.py

```diff
@@ -1130,25 +1132,50 @@ class LoginOTP(GenericViewSet):
 
     @transaction.atomic
     def generate(self, request, format=None):
-        response = {'exists': 0}
+        # response = {'exists': 0}
+        # serializer = serializers.OTPSerializer(data=request.data)
+        # serializer.is_valid(raise_exception=True)
+
+        # data = serializer.validated_data
+        # phone_number = data['phone_number']
+
+        # otp_obj = data.get('otp_obj')
+
+        # req_type = request.query_params.get('type')
+        # via_sms = data.get('via_sms', True)
+        # via_whatsapp = data.get('via_whatsapp', False)
+        # call_source = data.get('request_source')
+        # retry_send = request.query_params.get('retry', False)
+        # otp_message = OtpVerifications.get_otp_message(request.META.get('HTTP_PLATFORM'), req_type, version=request.META.get('HTTP_APP_VERSION'))
+        # otp_message = "test"
+        # send_otp(otp_message, phone_number, retry_send, via_sms=via_sms, via_whatsapp=via_whatsapp, call_source=call_source)
+        # if User.objects.filter(phone_number=phone_number, user_type=User.CONSUMER).exists():
+        #     response['exists'] = 1
```

Comment:
@KeelTech  I do not know whether this code is doing anything or not. so commented it out. let me know if to keep any